Automattic/pocket-casts-android
# Display Top 5 Shows Story

## Description

This adds top 5 shows story.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_3698

The behavior on small devices isn't specified. I opted for scrolling list with faded edges.

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the top 5 shows story and verify it.

## Screenshots or Screencast 

### Preview

![Screenshot_20241016-113827](https://github.com/user-attachments/assets/cde9d590-5cd7-42d2-ba7a-9c56b38467c0)

### Small device

https://github.com/user-attachments/assets/496f2cdf-cbd6-4d31-a997-79c9a1847cbf

### Animation

https://github.com/user-attachments/assets/635f3f64-ef28-411d-a0be-ee76bda3b33f

## Checklist
 
#### I have tested any UI changes...
